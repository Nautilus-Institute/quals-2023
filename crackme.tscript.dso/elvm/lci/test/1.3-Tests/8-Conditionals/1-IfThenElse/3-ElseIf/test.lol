HAI 1.3
	FAIL
	O RLY?
		YA RLY
			VISIBLE "1a"
		MEBBE BOTH OF WIN AN WIN
			VISIBLE "1b"
	OIC
	FAIL
	O RLY?
		YA RLY
			VISIBLE "2a"
		MEBBE BOTH OF WIN AN WIN
			VISIBLE "2b"
		NO WAI
			VISIBLE "2c"
	OIC
	FAIL
	O RLY?
		YA RLY
			VISIBLE "3a"
		MEBBE BOTH OF WIN AN FAIL
			VISIBLE "3b"
		NO WAI
			VISIBLE "3c"
	OIC
	FAIL
	O RLY?
		YA RLY
			VISIBLE "4a"
		MEBBE BOTH OF WIN AN FAIL
			VISIBLE "4b"
	OIC
	FAIL
	O RLY?
		YA RLY
			VISIBLE "5a"
		MEBBE EITHER OF FAIL AN FAIL
			VISIBLE "5b"
		MEBBE BOTH OF WIN AN FAIL
			VISIBLE "5c"
		MEBBE ANY OF FAIL AN FAIL AN FAIL AN FAIL AN WIN MKAY
			VISIBLE "5d"
		NO WAI
			VISIBLE "5e"
	OIC
KTHXBYE
