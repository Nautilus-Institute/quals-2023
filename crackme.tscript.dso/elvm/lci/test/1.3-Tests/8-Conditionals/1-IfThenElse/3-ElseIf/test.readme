This test checks that the operation of the else-if branch in an if/then/else
block works correctly.
