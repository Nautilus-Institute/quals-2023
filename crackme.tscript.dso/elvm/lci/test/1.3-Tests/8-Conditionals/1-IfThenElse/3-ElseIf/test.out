1b
2b
3c
5d
